src/Client/KeenIOClient.php

```diff
@@ -37,6 +37,12 @@
  * @method array funnel(string $eventCollection, array $args = array()) {@command KeenIO funnel}
  * @method array multiAnalysis(string $eventCollection, array $args = array()) {@command KeenIO multiAnalysis}
  * @method array extraction(string $eventCollection, array $args = array()) {@command KeenIO extraction}
+ * @method array createAccessKey(array $args = array()) {@command KeenIO createAccessKey}
+ * @method array listAccessKeys(array $args = array()) {@command KeenIO listAccessKeys+}
```

Comment:
why is there a `+` here ?

Reply:
i think it shows the difference between PR and Current Master branch